### PR TITLE
fix bug: crash when generate model with goctl.

### PR DESCRIPTION
**Pull request by 600ML**
situation: column name with line.

CREATE TABLE test (
id int NOT NULL AUTO_INCREMENT,
zh-cn text CHARACTER SET utf8 COLLATE utf8_general_ci COMMENT '中文简体',
PRIMARY KEY (id) USING BTREE,
) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;

### CODE REVIEW

**Review by 600ML — COMMENTED** [review]
虽然这样改，影响面最小，但是还是觉得ReplaceAll("-", "_")写到stringx.ToCamel()里更方便
只是我今天刚接触这个框架，学习中，还是谨慎点

File: tools/goctl/model/sql/gen/field.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package gen
 
 import (
+	"github.com/zeromicro/go-zero/tools/goctl/util/stringx"
```
Comment (kevwan) [inline tools/goctl/model/sql/gen/field.go:4]: imports need to be grouped.

File: tools/goctl/model/sql/gen/field.go [hunk 2]
```diff
@@ -38,7 +39,7 @@ func genField(table Table, field *parser.Field) (string, error) {
 	output, err := util.With("types").
 		Parse(text).
 		Execute(map[string]interface{}{
-			"name":       field.Name.ToCamel(),
+			"name":       stringx.From(field.Name.ReplaceAll("-", "_")).ToCamel(),
```
Comment (anqiansong) [inline tools/goctl/model/sql/gen/field.go:42]: Use https://github.com/zeromicro/go-zero/blob/6d9dfc08f99278dc02ecc774f83081573628365f/tools/goctl/util/string.go#L68 instead `ReplaceAll`

File: tools/goctl/model/sql/gen/insert.go [hunk 3]
```diff
@@ -30,7 +30,7 @@ func genInsert(table Table, withCache, postgreSql bool) (string, string, error)
 	expressionValues := make([]string, 0)
 	var count int
 	for _, field := range table.Fields {
-		camel := field.Name.ToCamel()
+		camel := stringx.From(field.Name.ReplaceAll("-", "_")).ToCamel()
```
Comment (anqiansong) [inline tools/goctl/model/sql/gen/insert.go:33]: Use https://github.com/zeromicro/go-zero/blob/6d9dfc08f99278dc02ecc774f83081573628365f/tools/goctl/util/string.go#L68 instead `ReplaceAll`

File: tools/goctl/model/sql/gen/update.go [hunk 4]
```diff
@@ -14,7 +14,7 @@ import (
 func genUpdate(table Table, withCache, postgreSql bool) (string, string, error) {
 	expressionValues := make([]string, 0)
 	for _, field := range table.Fields {
-		camel := field.Name.ToCamel()
+		camel := stringx.From(field.Name.ReplaceAll("-", "_")).ToCamel()
```
Comment (anqiansong) [inline tools/goctl/model/sql/gen/update.go:17]: Use https://github.com/zeromicro/go-zero/blob/6d9dfc08f99278dc02ecc774f83081573628365f/tools/goctl/util/string.go#L68 instead `ReplaceAll`

**Pull request merged**